src/meta/src/manager/sink_coordination/coordinator_worker.rs

```diff
@@ -724,20 +704,27 @@ impl CoordinatorWorker {
                     let SinkCommitCoordinator::TwoPhase(coordinator) = &mut coordinator else {
                         unreachable!("should be two-phase commit coordinator");
                     };
-                    let two_phase_handler = two_phase_handler.as_mut().expect("should exist");
                     let start_time = Instant::now();
-                    let commit_res = run_future_with_periodic_fn(
-                        coordinator.commit(epoch, metadata, schema_change),
-                        Duration::from_secs(5),
-                        || {
+                    let commit_fut = async {
+                        if !metadata.is_empty() {
```

Comment:
Instead of checking on empty `Vec<u8>`, shall we use `Option<Vec<u8>>` to represent whether there is metadata to commit? Our meta store table should support storing and retrieving `None`.

Reply:
Fortunately, the migration file is not included in any release yet. We can remove the `not_null` in the migration file directly.